src/engine/interfaces.ts

```diff
@@ -18,7 +18,14 @@ export interface StepArguments {
   distributableRewards: BigNumber;
 }
 
+export enum EPaymentType {
+  Delegator = "delegator_payment",
+  FeeIncome = "fee_income_payment",
+  BondReward = "bond_reward",
+}
```

Comment:
What about nice names? (`Bond Reward`, `Fee Income Payment`...) This output is primarily for users so I would go with nice names if possible.